ui-admin/src/study/participants/datarepo/DatasetDashboard.tsx

```diff
@@ -0,0 +1,165 @@
+      sorting
+    },
+    onSortingChange: setSorting,
+    getCoreRowModel: getCoreRowModel(),
+    getSortedRowModel: getSortedRowModel(),
+    debugTable: true
+  })
+
+  const contentHeaderStyle = {
+    padding: '1em 0 0 1em',
+    borderBottom: '1px solid #f6f6f6'
+  }
+
+  const loadData = async () => {
+    try {
+      //Fetch dataset details
+      const datasetDetails = await Api.listDatasetsForStudyEnvironment(
+        studyEnvContext.portal.shortcode,
+        studyEnvContext.study.shortcode,
+        studyEnvContext.currentEnv.environmentName)
+      const datasetDetailsResponse = await datasetDetails.json()
+      setDatasetDetails(datasetDetailsResponse.find((dataset: { datasetName: string }) =>
+        dataset.datasetName === datasetName))
+
+      //Fetch dataset job history
+      const datasetJobHistory = await Api.getJobHistoryForDataset(
+        studyEnvContext.portal.shortcode,
+        studyEnvContext.study.shortcode,
+        studyEnvContext.currentEnv.environmentName,
+        datasetName)
+      const datasetJobHistoryResponse = await datasetJobHistory.json()
+      setDatasetJobHistory(datasetJobHistoryResponse)
+
+      setIsLoading(false)
+    } catch (e) {
+      Store.addNotification(failureNotification(`Error loading dataset information`))
+    }
+  }
+
+  useEffect(() => {
+    loadData()
+  }, [])
+  return <div className="container-fluid py-3">
+    <h1 className="h3">Terra Data Repo</h1>
+    <Link to={getDatasetListViewPath(currentEnvPath)} className="mx-2">
+      <FontAwesomeIcon icon={faArrowLeft}/> Back to dataset list
+    </Link>
+    <LoadingSpinner isLoading={isLoading}>
+      <div className="col-12 p-3">
+        <ul className="list-unstyled">
+          <li className="bg-white my-3">
+            <div style={contentHeaderStyle}>
+              <h6>Dataset Details</h6>
+            </div>
+            <div className="flex-grow-1 p-3">
+              <div className="form-group">
+                <div className="form-group-item">
+                  <label>Dataset Name:</label> { datasetDetails?.datasetName }
+                  <br/>
+                  <label>Dataset ID:</label> { datasetDetails?.datasetId }
+                  <br/>
+                  <label>Created Date:</label> { instantToDefaultString(datasetDetails?.createdAt) }
+                  <br/>
+                  <label>Last Successful Export:</label> { instantToDefaultString(datasetDetails?.lastExported) }
+                </div>
+                <br/>
+                <a href={`https://jade.datarepo-dev.broadinstitute.org/datasets/${datasetDetails?.datasetId}`}
+                  target="_blank">View dataset in Terra Data Repo <FontAwesomeIcon icon={faExternalLink}/></a>
+              </div>
+            </div>
+          </li>
+          <li className="bg-white my-3">
+            <div style={contentHeaderStyle}>
+              <h6>Export History</h6>
+            </div>
+            <table className="table table-striped">
```

Comment:
since you're just using the basics, you can replace all the table markup with a call to `basicTableLayout` in tableUtils

Reply:
Sweet, done in 64eb80a3